src/main/java/racingcar/common/util/RandomUtils.java

```diff
@@ -0,0 +1,12 @@
+package racingcar.common.util;
+
+import java.util.Random;
+
+public class RandomUtils {
+    private RandomUtils() {}
+    private static int DEFAULT_RANDOM_SIZE = 10;
+
```

Comment:
클래스 변수 -> 일반 변수 -> 생성자 
순으로 선언해주는 것이 [컨벤션](https://myeonguni.tistory.com/1596) 입니다! 😄 
```suggestion
public class RandomUtils {
    private static int DEFAULT_RANDOM_SIZE = 10;

    private RandomUtils() {}
```

Reply:
여태 순서를 크게 고려하지 않았었는데 이번 기회에 잘 기억해두도록 하겠습니다.
감사합니다~